articles/app-service/app-service-key-vault-references.md

```diff
@@ -53,6 +53,9 @@ If your vault is configured with [network restrictions](../key-vault/general/ove
 > [!NOTE]
 > Windows container currently does not support Key Vault references over VNet Integration.
 
+> [!NOTE]
+> Please note that in case of keyvault references the origin IP might not be one of the outbound IPs, that is why it is highly recommended that if you wish to restrict your Key Vault behind an IP Firewall, to use vnet integration either at the app level or through an ASE.
+
```

Comment:
I would suggest taking this out of a note and instead add something to the first paragraph of this section on [line 31](https://github.com/MicrosoftDocs/azure-docs/pull/108209/files#diff-681c7c190dfe898a0ead1bdd0286696369904474d686180cdf789008c0a85c57R31). Perhaps we could just add a sentence like "Vaults should not depend upon the app's public outbound IPs, as the origin IP of the secret request could be different. Instead, the vault should be configured to accept traffic from a virtual network used by the app."